StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+FormSpec.swift

```diff
@@ -7,38 +7,111 @@
 //
 
 import Foundation
+import StripeCore
 @_spi(STP) import StripeUICore
 
 extension PaymentSheetFormFactory {
-
     func specFromJSONProvider(provider: FormSpecProvider = FormSpecProvider.shared) -> FormSpec? {
         guard let paymentMethodType = PaymentSheet.PaymentMethodType.string(from: paymentMethod) else {
             return nil
         }
         return provider.formSpec(for: paymentMethodType)
     }
 
-    func makeFormElementFromSpec(spec: FormSpec) -> FormElement {
+    func makeFormElementFromSpec(
+        spec: FormSpec,
+        additionalElements: [Element] = []
+    ) -> PaymentMethodElementWrapper<FormElement> {
```

Comment:
Hm.. it's not immediately obvious to me why we are needing to wrap the FormElement in a PaymentMethodElementWrapper. I'm guessing it's an elegant solution to a problem, but not seeing it -- Could you please explain why we need to do so?